### ISSUE TITLE
Error message: Failed to collect Storybook metadata

### ISSUE THREAD

**Issue by domyen** [chromaui/chromatic-cli#324]
```
Error: ✖ Failed to collect Storybook metadata
Could not find a supported Storybook viewlayer package. Make sure one is installed, or set CHROMATIC_STORYBOOK_VERSION.
at...
```

Users often get this message because Storybook is not installed as a dependency in their environment. This can happen in CI or locally if they've just cloned the project.

**Strawman** 
Be more direct about a possible solution.

    Error: ✖ Failed to collect Storybook metadata
    Could not find a supported Storybook view layer package. This can happen if the dependency hasn't been installed in your environment yet. Run `npm install` or `yarn` to make sure Storybook is installed. 
    
    If Storybook is installed, you can also set CHROMATIC_STORYBOOK_VERSION.
    at...

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1199921881886840/1200263242480621) by [Unito](https://www.unito.io)

**Comment by domyen** [comment]
This is resolved by PR #319 and related to #317. Close when those are merged.

**Comment by BeeeQueue** [comment]
We have an internal package that installs all relevant storybook packages as dependencies and contains configuration and utilities.

```
root
  node_modules
    storybook-config
      node_modules
        chromatic
        storybook
```

We're getting the error telling us storybook isn't installed since it's in a nested folder.

Does this solution handle this?

**Comment by productfrontenddeveloper** [comment]
some problem
![](https://user-images.githubusercontent.com/38668796/118359449-3427bb00-b58c-11eb-9511-ee858d2c491b.png)

**Comment by jackmccloy** [comment]
@domyen just wanted to give you a heads up that is also seems to happen if you use yarn v1 workspaces and install packages at the workspace root, rather than in individual projects (which isn't exactly a great practice, but can be convenient for projects that use create-react-app).

the workaround is to list your current version of storybook as a dependency or devdependency in the project itself, but it'd be lovely if it worked w/o having to do this (as it did a month or so ago)

**Comment by tmeasday** [comment]
Released 5.8.1 which reverts to something similar to the earlier behaviour, please try it out.
